libs/api/ebpf_api.cpp

```diff
@@ -53,9 +54,13 @@ ebpf_api_terminate()
     }
 }
 
-int
-create_map_function(
-    uint32_t type, uint32_t key_size, uint32_t value_size, uint32_t max_entries, ebpf_verifier_options_t)
+uint32_t
+ebpf_api_create_map(
+    _In_ const ebpf_map_type_t type,
+    _In_ const uint32_t key_size,
+    _In_ const uint32_t value_size,
+    _In_ const uint32_t max_entries,
```

Comment:
```suggestion
    const ebpf_map_type_t type,
    const uint32_t key_size,
    const uint32_t value_size,
    const uint32_t max_entries,
```
Value types should never have SAL annotation, fix throughout